src/View/AppView.php

```diff
@@ -49,4 +51,22 @@ public function initialize() : void
         $this->getTwig()
             ->addExtension(new BeditaTwigExtension());
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * Retrieve custom view `element` from configuration mappings.
+     *
+     * If `Elements.{module_name}.{element_name}` exists in configuration a custom element is loaded
+     */
+    protected function _getElementFileName($name, $pluginCheck = true)
+    {
+        $module = $this->get('currentModule');
+        $custom = Configure::read(sprintf('Elements.%s', Hash::get($module, 'name', '')));
```

Comment:
Are we sure `currentModule` is always defined and is an array? `Hash::get()` require the first argument to be an array or  instance of `\ArrayAccess`.

Maybe better `$module = $this->get('currentModule', []);` to have a default.

Reply:
It's an array, but it could be `null` - more checks are needed, including the one you're proposing